src/components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,3 @@
+export const HomePage: React.FC = () => (
+  <h1 className="title">Home Page</h1>
+);
```

Comment:
Well, it's strange why linter didn't warn you about it, but we need to import React to use JSX syntax, also FC type can be imported from react lib too